src/shared/components/ChannelPreview/ChannelPreviewBase.tsx

```diff
@@ -1,37 +1,73 @@
 import React from 'react'
-import { AvatarContainer, Info, InnerContainer, OuterContainer } from './ChannelPreviewBase.style'
+import {
+  Anchor,
+  AvatarContainer,
+  Info,
+  InnerContainer,
+  NameHeader,
+  OuterContainer,
+  StyledAvatar,
+  VideoCount,
+  VideoCountContainer,
+} from './ChannelPreviewBase.style'
 import Placeholder from '../Placeholder'
+import { CSSTransition, SwitchTransition } from 'react-transition-group'
+import { transitions } from '@/shared/theme'
 
 type ChannelPreviewBaseProps = {
-  avatarNode?: React.ReactNode
-  nameNode?: React.ReactNode
+  avatarURL?: string
```

Comment:
Could we rename that to `avatarUrl`? We used `URL` in queries before but at some point we switch to `Url`